Sources/SwifterSwift/CoreAnimation/CATransform3DExtensions.swift

```diff
@@ -0,0 +1,281 @@
+    @inlinable
+    public init(from decoder: Decoder) throws {
+        var container = try decoder.unkeyedContainer()
+        self.init(m11: try container.decode(CGFloat.self),
+                  m12: try container.decode(CGFloat.self),
+                  m13: try container.decode(CGFloat.self),
+                  m14: try container.decode(CGFloat.self),
+                  m21: try container.decode(CGFloat.self),
+                  m22: try container.decode(CGFloat.self),
+                  m23: try container.decode(CGFloat.self),
+                  m24: try container.decode(CGFloat.self),
+                  m31: try container.decode(CGFloat.self),
+                  m32: try container.decode(CGFloat.self),
+                  m33: try container.decode(CGFloat.self),
+                  m34: try container.decode(CGFloat.self),
+                  m41: try container.decode(CGFloat.self),
+                  m42: try container.decode(CGFloat.self),
+                  m43: try container.decode(CGFloat.self),
+                  m44: try container.decode(CGFloat.self))
+    }
+
+    /// Encodes this value into the given encoder.
+    ///
+    /// If the value fails to encode anything, encoder will encode an empty keyed container in its place.
+    ///
+    /// This function throws an error if any values are invalid for the given encoder’s format.
+    /// - Parameter encoder: The encoder to write data to.
+    @inlinable
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.unkeyedContainer()
+        try container.encode(m11)
+        try container.encode(m12)
+        try container.encode(m13)
+        try container.encode(m14)
+        try container.encode(m21)
+        try container.encode(m22)
+        try container.encode(m23)
+        try container.encode(m24)
+        try container.encode(m31)
+        try container.encode(m32)
+        try container.encode(m33)
+        try container.encode(m34)
+        try container.encode(m41)
+        try container.encode(m42)
+        try container.encode(m43)
+        try container.encode(m44)
+    }
+
+    // swiftlint:enable missing_swifterswift_prefix
+
+}
+
+// MARK: - Initializers
+public extension CATransform3D {
+
+    /// SwifterSwift: Returns a transform that translates by `(tx, ty, tz)`.
+    /// - Parameters:
+    ///   - tx: x-axis translation
+    ///   - ty: y-axis translation
+    ///   - tz: z-axis translation
+    @inlinable
+    init(translationX tx: CGFloat, y ty: CGFloat, z tz: CGFloat) { // swiftlint:disable:this identifier_name
+        self = CATransform3DMakeTranslation(tx, ty, tz)
+    }
+
+    /// SwifterSwift: Returns a transform that scales by `(sx, sy, sz)`.
+    /// - Parameters:
+    ///   - sx: x-axis scale
+    ///   - sy: y-axis scale
+    ///   - sz: z-axis scale
+    @inlinable
+    init(scaleX sx: CGFloat, y sy: CGFloat, z sz: CGFloat) { // swiftlint:disable:this identifier_name
```

Comment:
This is just a comment for maybe another PR, we could add identifiers like : i, j, x, y, z as exceptions in for this rule in the yaml :)